codeHF/dpl-config_run3.json

```diff
@@ -43,6 +43,9 @@
         "doCutQuality": "true",
         "useIsGlobalTrack": "false",
         "useIsGlobalTrackWoDCA": "false",
+        "useIsGlobalTrackForSoftPion": "false",
+        "useIsGlobalTrackWoDCAForSoftPion": "true",
+        "useIsQualityTrackITSForSoftPion": "false",
```

Comment:
`useIsGlobalTrackWoDCA` and `useIsGlobalTrackWoDCAForSoftPion` should both be `true`.